charts/auth-layer-server/templates/realm-configmap.yaml

```diff
@@ -0,0 +1,2530 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: keycloak-realm-config
+data:
+  realm.json: |
```

Comment:
Q: this is a very large json value.
Maybe not for this PR but could this be split into more than one file and mounted or does it have to be one?
Not sure how this is readable.

Reply:
Yes, indeed is a large json, this is the export file that KeyCloak produces when we Export a given Realm, for backup or sharing purposes.
Also not sure if we can split it, most likely not, that said, this file is not meant to be read or modify manually, is expected to be `imported` as is and then visualized on the GUI.

Reply:
Should probably have such a comment somewhere then.